chapters/en/chapter6/pre-trained_models.mdx

```diff
@@ -0,0 +1,330 @@
+that capture a particular speaker's voice characteristics.
+
+<Tip>
+
+Speaker embeddings is a method of representing a speaker's identity in a compact way, as a vector of 
+fixed size, regardless of the length of the utterance. These embeddings capture essential information about a speaker's 
+voice, accent, intonation, and other unique characteristics that distinguish one speaker from another. Such embeddings can 
+be used for speaker verification, speaker diarization, speaker identification, and more. 
+The most common techniques for generating speaker embeddings include:
+
+* I-Vectors (identity vectors): I-Vectors are based on a Gaussian mixture model (GMM). They represent speakers as low-dimensional fixed-length vectors derived from the statistics of a speaker-specific GMM, and are obtained in unsupervised manner. 
+* X-Vectors: X-Vectors are derived using deep neural networks (DNNs) and capture frame-level speaker information by incorporating temporal context. 
+
+[X-Vectors](https://www.danielpovey.com/files/2018_icassp_xvectors.pdf) are a state-of-the-art method that shows superior performance 
+on evaluation datasets compared to i-vectors. The deep neural network is used to obtain X-Vectors: it trains to discriminate 
+between speakers, and maps variable-length utterances to fixed-dimensional embeddings. 
+
+</Tip>
+
+Let's load such a speaker embedding from a dataset on the Hub. The embeddings 
+were obtained from the [CMU ARCTIC dataset](http://www.festvox.org/cmu_arctic/) using 
+[this script](https://huggingface.co/mechanicalsea/speecht5-vc/blob/main/manifest/utils/prep_cmu_arctic_spkemb.py), but 
+any X-Vector embedding should work.
+
+```python
+from datasets import load_dataset
+
+embeddings_dataset = load_dataset("Matthijs/cmu-arctic-xvectors", split="validation")
+
+import torch
+
+speaker_embeddings = torch.tensor(embeddings_dataset[7306]["xvector"]).unsqueeze(0)
+```
+
+The speaker embedding is a tensor of shape (1, 512). This particular speaker embedding describes a female voice.
+
+At this point we already have enough inputs to generate a log mel spectrogram as an output, you can do it like this:
+
+```python
+spectrogram = model.generate_speech(inputs["input_ids"], speaker_embeddings)
+```
+
+This outputs a tensor of shape (140, 80) containing a log mel spectrogram. The first dimension is the sequence length, and 
+it may vary between runs as the speech decoder pre-net always applies dropout to the input sequence. This adds a bit of 
+random variability to the generated speech.
+
+However, if we are looking to generate speech waveform, we need to specify a vocoder to use for the spectrogram to waveform conversion.
+In theory, you can use any vocoder that works on 80-bin mel spectrograms. Conveniently, 🤗 Transformers offers a vocoder 
+based on HiFi-GAN. Its weights were kindly provided by the original authors of SpeechT5.
+
+<Tip>
```

Comment:
I think tips need to be in HTML to render correctly - the paragraphs aren't splitting on line (so let's just decorate each paragraph with `<p> Text... </p>`

Reply:
Tried adding html, doesn't seem to do anything. I think it's not the end of the world, though, if it stays as is.